src/m365/context/commands/context-remove.ts

```diff
@@ -12,8 +23,84 @@ class ContextRemoveCommand extends ContextCommand {
     return 'Removes the CLI for Microsoft 365 context in the current working folder';
   }
 
-  public async commandAction(logger: Logger): Promise<void> {
-    await this.removeContextInfo(logger);
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    if (args.options.confirm) {
+      await this.removeContextInfo(logger);
+    }
+    else {
+      const result = await Cli.prompt<{ continue: boolean }>({
+        type: 'confirm',
+        name: 'continue',
+        default: false,
+        message: `Are you sure you want to remove the Microsoft Power App ${args.options.name}?`
+      });
+
+      if (result.continue) {
+        await this.removeContextInfo(logger);
```

Comment:
Like I suggested earlier, let's replace all references to `contextInfo` with just `context` so that we avoid confusing folks who associate contextInfo with SharePoint's request digest